common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -1,10 +1,16 @@
 {
   "advancement.estrogen.balls.description": "Was passiert, wenn man den Schleim aus dem Ball presst??",
   "advancement.estrogen.balls.title": "Hehe",
+  "advancement.estrogen.cookie_jar.description": "Diese Mod braucht Kekse um zu funktionieren. Bitte lege diese in das Glass.",
+  "advancement.estrogen.cookie_jar.title": "Akzeptieren & Weiter",
+  "advancement.estrogen.estrogen_dealer.description": "Ich bin derjenige der klopft",
+  "advancement.estrogen.estrogen_dealer.title": "Östrogen Händler",
```

```suggestion
  "advancement.estrogen.estrogen_dealer.title": "Östrogenhändler",
```
Just like the already existing "Östrogenpille", the trader would be named with a composite word.